lib/OpenQA/CacheService/Response.pm

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2019 SUSE LLC
+# Copyright (C) 2018 SUSE LLC
```

Comment:
Git makes it look like this has been changed from 2019 to 2018. Maybe you add both years.